docs/process/releasing-updates.md

```diff
@@ -47,7 +47,7 @@ Here's what you should do next:
 To walk through this:
 
  - the script works out the next version from what was previously published, based on the channel
- - you should ensure the `version` in `app/package.json` is set to the new version
+ - you should ensure the `version` in `app/package.json` is set to the new version and follows the [semver format](https://semver.org/)
```

Comment:
Perhaps make this more explicit?

> and follows the [semver format](https://semver.org/) of `major.minor.patch`?

Reply:
or an example. we should also provide the example for production v beta v test (since we've somewhat settled on a pattern for beta/test as well)